source/standards/programming-languages.html.md.erb

```diff
@@ -77,30 +77,26 @@ production.
 
 [PEP373]: https://www.python.org/dev/peps/pep-0373/
 
-### Go
+### Languages we do not use for new projects
+
+#### Scala
+
+We used Scala in the early days of GDS. GOV.UK Licensing is the only remaining
+application written in Scala but we've found it hard to support because of a lack
+of skills in GDS. Do not use Scala for new projects.
+
+#### Go
 
 Go is no longer a core backend development language in GDS.
 
 The only Go service currently in production operation is the [GOV.UK
-router][router], and it is the core language for [Cloud
-Foundry](https://www.cloudfoundry.org/), which GOV.UK PaaS uses, although
-GOV.UK PaaS is being [decommissioned][pass-decom]. As such, the knowledge and
-experience of building and running services in Go is small and decreasing.
+Foundry](https://www.cloudfoundry.org/), which was used by GOV.UK PaaS.
```

Comment:
You may want to keep the precious copy? Router is still used, right? And CloudFoundry is no-mo, along the paas-decom.

Reply:
I'm not sure what's happened with this commit, but this paragraph makes no sense. I'm assuming this is supposed to say GOV.UK Router, and there's no need to mention PaaS:

> the GOV.UK Foundry, which was used by GOV.UK PaaS